example_byteslice_littleendian_test.go

```diff
@@ -9,3 +9,10 @@ func ExampleRSet() {
 	fmt.Printf("%x\n", RSet(data, setData))
 	// Output: bada55
 }
+
+func ExampleRUnset() {
+	data := []byte{0xDA, 0x99, 0xBA}
+	unsetData := []byte{0xAD, 0x11, 0xAB}
+	fmt.Printf("%x\n", RUnset(data, unsetData))
+	// Output: 8811aa
```

Comment:
For the `ExampleUnset`, Can you use the output `bada55` and create the **data**, and **unsetData** to provide this output.